app/models/vacols/case_docket.rb

```diff
@@ -144,8 +144,11 @@ def self.counts_by_priority_and_readiness
       select count(*) N, PRIORITY, READY
```

Comment:
Optional:
```suggestion
      select count(*) COUNT, PRIORITY, READY
```

Would have to update: https://github.com/department-of-veterans-affairs/caseflow/blob/6287312416efb2e55988a3d1bf5d69fbfefb235d/app/models/dockets/legacy_docket.rb#L20
and some tests.

Using 'N' seems to be a pattern: https://github.com/department-of-veterans-affairs/caseflow/blob/96fa3863bd31850bc7deb2b8816f70c3d0cc6926/app/models/vacols/case_assignment.rb#L149